### PR TITLE
Added an alphabetical sort button to developer's page.

### PR DESCRIPTION
**Pull request by jessamarie**

### CODE REVIEW

**Comment by agundy** [comment]
Did you test this with pagination?

File: client/app/user/user.html [hunk 1]
```diff
@@ -47,7 +51,7 @@
       </div>
     </div>
     <div class="row image-row">
-        <div ng-repeat="user in users | filter:searchString | startFrom:currentPage*pageSize | limitTo:pageSize">
+        <div ng-repeat="user in users | filter:searchString | startFrom:currentPage*pageSize | limitTo:pageSize |  orderBy: sortOrder">
```
Comment (agundy) [inline client/app/user/user.html]: Watch out for the ordering here! you are limiting the length of the list and then sorting, sorting should happen first.

File: client/app/user/user.controller.js [hunk 2]
```diff
@@ -23,6 +24,14 @@ angular.module('observatory3App')
       $scope.users = $scope.currentUsers;
     }
 
+    $scope.toggleSortOrder = function(){
+      if($scope.sortOrder === '-name'){
```
Comment (agundy) [inline client/app/user/user.controller.js]: Nitpick but i like spaces between if and parentheses

**Comment by jessamarie** [comment]
I didn't think to test this with pagination originally, but after moving the sort filter it's working on my end.

**Comment by agundy** [comment]
All good! this is why we do code reviews :smile:

**Comment by kmcnellis** [comment]
How would this work with the back button? Maybe we should add the sorting parameter to the url as well.

**Comment by jessamarie** [comment]
@kmcnellis I'm on it!!

**Pull request merged**